test/test_core.py

```diff
@@ -2988,7 +2988,7 @@ class Foo {
           EM_ASM( out("Constructing lib object.") );
         }
       };
-      Foo global;
```

Comment:
`static` might be another option here IIUC? no preference though.

Reply:
This was the fix that was applied to `test_dlfcn_basic` which is a copy of this code, so this change keeps them in sync.